Content.Server/Chemistry/TileReactions/FlammableTileReaction.cs

```diff
@@ -18,7 +18,7 @@ public FixedPoint2 TileReact(TileRef tile, ReagentPrototype reagent, FixedPoint2
             if (reactVolume <= FixedPoint2.Zero || tile.Tile.IsEmpty)
                 return FixedPoint2.Zero;
 
-            var atmosphereSystem = EntitySystem.Get<AtmosphereSystem>();
+            var atmosphereSystem = IoCManager.Resolve<IEntityManager>().System<AtmosphereSystem>();
 
```

Comment:
I'd rather just leave these ones in so reagents get done properly instead of making funny number go down as they're similar in terms of badness.

Reply:
its just tilereactions afaict, ill just do a proper solution for them

Reply:
removed these from all non-comp VV shit